### PR TITLE
R 4 4 0 updates

### PR DESCRIPTION
**Pull request by pbbgss**

### CODE REVIEW

File: renv.lock
Comment (tpin13) [inline renv.lock:1]: Checked that new packages install correctly on R 4.4.0, and that content.Rmd code runs as expected with new packages.

File: rmd_files/content.Rmd
Comment (tpin13) [inline rmd_files/content.Rmd:1]: New version works well and addresses issue #86. Not sure what the problem with old code was though.

**Review by tpin13 — APPROVED** [review]
Have checked content code still runs and that changes to && work correctly.

File: rmd_files/content.Rmd
Comment (pbbgss) [reply rmd_files/content.Rmd:1]: This will result in an error as of R 4.2.0:
```R
if (TRUE & c(TRUE, FALSE)) {
}
```

Comment (tpin13) [reply rmd_files/content.Rmd:1]: Ah, thanks for the explanation

**Pull request merged**